node/pkg/logscribeconsumer/app.go

```diff
@@ -108,7 +128,19 @@ func (a *App) processBatch(ctx context.Context) error {
 				break loop
 			}
 		}
-		return bulkCopyLogEntries(ctx, batch)
+		batchLen := len(batch)
+		errors := []error{}
+		for i := 0; i < batchLen; i += ProcessLogsBatchSize {
```

Comment:
Think you added based on coderabbit feedback, but this batch processing seems unnecessary 🤔 it is good to have, but probably I'm sure it'll be okay without this

If you insist you can keep it

Reply:
yea i think it won't hurt in case if we accidentally get spammed with errors. And if not, the loop should finish in one iteration